benchmarks/nsinker_spherical_shell/README.md

```diff
@@ -0,0 +1,21 @@
+The Spherical Shell NSinker benchmark
+=====================================
+
+This folder contains the "Spherical Shell NSinker" benchmark, which is
+loosely based on the NSinker benchmark defined in
+
+Weighted BFBT Preconditioner for Stokes Flow Problems with Highly
+Heterogeneous Viscosity
+Johann Rudi, Georg Stadler, Omar Ghattas
+https://doi.org/10.1137/16M108450X
+
+but adapted to a spherical shell geometry. See the "nsinker"
+benchmark in this repository for the version in a unit box.
+
+We consider a spherical shell with inner radius 0.54 and outer radius
+1.0 with N spherical, high-viscosity, high-density sinking spheres at
+random locations in the domain. Additionally, a transition zone with
```

Comment:
This sounds like they would be different every time, or at least like ASPECT would create some pseudo-random numbers, but these are pre-generated locations from a paper. It would be good to make that clear.

Reply:
fixed.